### PR TITLE
Portfolio: deskstop version

### PR DESCRIPTION
**Pull request by blessing-michael**
# Portfolio: Desktop version

In this project I;
- Used media queries to ensure the website displays correctly on desktop screens.
- Ensured there are [no linter errors](https://github.com/microverseinc/linters-config).
- Used CSS Grid and Flexbox to lay elements on the page.

### CODE REVIEW

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @blessing-michael, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

- [ ] It is required to stick to the figma design as much as possible. kindly work on the alignment of the project information beside each of the project image. Tech lists of each project are not properly aligned. Kindly check the two screenshots below and work your design alongside the figma design.

### Your Design
your-work-card

### Figma Design
figma-work-card

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbdusSattar-70 — COMMENTED** [review]
## Invalid CRR
Hi @blessing-michael , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that didn't pass all the linters, Please make sure to pass the linters before submitting the re-review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -20,143 +19,154 @@
...
+              </p>
+  
+              <div class="tech-highligt">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
               </div>
-
-              <button class="project-btn">See project</button>
+              <button class="tech-project-btn" type="submit">See project</button>
```
Comment (roshan-bajgain) [inline index.html:117]: - [ ] You did great work but while make changes on previous reviewer you implement the wrong design. According to the project requirement the page is divided half of it contains the project's picture and the other half contains the project's description. Please see image below for reference.
- [ ] Please implement with all the cards.
### Figma design
figma

### Your design
his

Comment (roshan-bajgain) [inline index.html:117]: - [ ] [Optional] I would suggest you please implement the active, hover and pressed effect on you buttons it make good user experience. Please implement with all the buttons
### Design
his

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @blessing-michael, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
- [ ] I know everyone is facing the `webhint` problem. Hope it will as soon as possible.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — APPROVED** [review]
Hi @blessing-michael, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

close

## Highlights

- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 
- [ ] No Linter Errors :green_circle: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**